modules/persistent-storage-csi-vsphere-increase-max-vols-per-node-procedure.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-increase-max-vols-per-node_{context}"]
+= Increasing the maximum allowable volumes per node for vSphere
+
+.Prerequisites
```

Comment:
As prerequisite customers need to set pvscsiCtrlr256DiskSupportEnabled to true in vCenter.
We need to document it but be VERY careful with the wording.
This is NOT officially supported by VMware and this is a global vCenter setting so even if customers want to try it on say a OCP test cluster, this setting will affect the entire vCenter which could also host production workloads.
@gnufied @RomanBednar

Reply:
@lpettyjo There was a [thread](https://github.com/openshift/openshift-docs/pull/91472/files#r2054190991) opened for this and then flagged as "Done", but I don't see this prerequisite note yet - PTAL.

Reply:
@gcharot @gnufied @RomanBednar I searched on this parameter in vSphere docs and didn't find anything, so I don't have a specific place to point users to. Maybe we need to discuss how to approach this due to its sensitive nature.

Reply:
Yeah, it is not documented anywhere in a public facing doc from Broadcom. That is the rub. @gcharot should we document our own stuff or ask customer to talk to Broadcom?

Reply:
I think we should mention the parameter name but clearly mention that this is not fully supported by VMware as well as pointing out this is a cluster wide option.

Reply:
Done.